lib/task_helpers.rb

```diff
@@ -17,7 +17,11 @@ def self.rebuild_index(alias_name=nil)
       index_name = create_index()
 
       [Comment, CommentThread].each do |model|
-        model.import(index: index_name)
+        model.import(index: index_name, batch_size: 100) do |response|
+            response['items'].select { |i| i['index']['error'] }.each do |item|
+                LOG.info item['index']['error']
```

Comment:
1. Is there a way for you to force a real error to test this?
2. I was thinking we might get an error count, and if it is > 0 then throw an exception at the end with the count and user friendly message.